### PR TITLE
website-django-changes

### PR DESCRIPTION
**Pull request by sumit-haptik**
New google doc for careers - https://docs.google.com/spreadsheets/d/14v7kHDyRezoMdfL0Lei3dAfnox6WUmet-0gHezoj4vc/edit#gid=0

### CODE REVIEW

File: js/career.js [hunk 1]
```diff
@@ -42,6 +42,12 @@ function applicationSubmit() {
       document.getElementById("email-address").focus();
       alert("Please fill the email address field");
       return false;
+    }else  {
+      var reg = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
+      if(reg.test(email_address) == false){
+        alert('Please enter valid EMail Address');
```
Comment (overthinker29) [inline js/career.js]: email *

File: js/app.js [hunk 2]
```diff
@@ -72,6 +72,12 @@ function partnersForm() {
         document.getElementById("email-address").focus();
         alert("Please fill the email field");
         return false;
+    }else{
+        var reg = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
+        if(reg.test(e_mail) == false){
+            alert('Please enter valid EMail Address');
```
Comment (overthinker29) [inline js/app.js]: email *

File: openingsApp/openings/tasks.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        if body:
+            err_msg += "\nSheet Details:\n"
+            err_msg += body
+        err_msg += "\nError Details: " + str(e)
+        err_email = EmailMessage(err_subject,
+                                 err_msg,
+                                 const_settings.kDEFAULT_FROM_REPORT_EMAIL,
+                                 ["errors@haptik.co"])
+        err_email.send()
+        logger.exception('<================= Archies + %s' % e)
```
Comment (overthinker29) [inline openingsApp/openings/tasks.py]: remove this

File: openingsApp/openingsApp/celery.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Stdlib Imports
```
Comment (overthinker29) [inline openingsApp/openingsApp/celery.py]: @pawantejwani @ranvijayj  Can one of you review this config made by Sumit for the website?

File: openingsApp/start_celery.sh [hunk 5]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (overthinker29) [inline openingsApp/start_celery.sh]: @pawantejwani @ranvijayj Same here...

Comment (ranvijayj) [inline openingsApp/start_celery.sh]: this looks fine.

**Pull request merged**